pkg/controller/vsphere/util.go

```diff
@@ -0,0 +1,82 @@
+package vsphere
+
+import (
+	"context"
+	"fmt"
+
+	"gopkg.in/gcfg.v1"
+
```

Comment:
extra spaces here

Reply:
I intentionally separated it from the k8s related imports.